test/unit/quantum_program/test_samplex_item.py

```diff
@@ -0,0 +1,156 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests the ``SamplexItem`` class."""
+
+import numpy as np
+
+from samplomatic import build, Twirl, InjectNoise
+
+from qiskit.circuit import QuantumCircuit, Parameter
+from qiskit.quantum_info import PauliLindbladMap
+
+from qiskit_ibm_runtime.quantum_program.quantum_program import SamplexItem
+
+
+from ...ibm_test_case import IBMTestCase
+
+
+class TestSamplexItem(IBMTestCase):
+    """Tests the ``SamplexItem`` class."""
+
+    def test_samplex_item(self):
+        """Test ``SamplexItem`` for a valid input."""
+        circuit = QuantumCircuit(2)
+        with circuit.box(annotations=[Twirl()]):
+            circuit.rx(Parameter("p"), 0)
+            circuit.cx(0, 1)
+        with circuit.box(annotations=[Twirl()]):
+            circuit.measure_all()
+
+        template_circuit, samplex = build(circuit)
+
+        parameter_values = np.array([[[1], [2]], [[3], [4]], [[5], [6]]])
+        samplex_shape = (30, 1, 2)
```

Comment:
I'd actually prefer this case to trigger an error, but this discussion is out of scope here.